rllib/algorithms/tests/test_worker_failures.py

```diff
@@ -1,15 +1,40 @@
+from collections import defaultdict
```

Comment:
We have to be careful here. Artur just moved these tests into rllib/tests/..., I believe.

Reply:
ok, I will sync with him